frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -792,7 +792,7 @@ function ReaderFooter:textOptionTitles(option)
         reclaim_height = _("Reclaim bar height from bottom margin"),
         bookmark_count = T(_("Bookmark count (%1)"), symbol_prefix[symbol].bookmark_count),
         page_progress = T(_("Current page (%1)"), "/"),
-        page_remaining = T(_("Pages remaining (%1)"), "-"),
+        pages_left_book = T(_("Pages remaining (%1)"), "-"),
```

Comment:
May be `Pages left in book` then.

Reply:
There's no maybe. The distinction between pages left and pages remaining cannot be chapter vs book. ;-)

Reply:
Or rather, there's no distinction at all. Those are synonyms.